lib/response/config.ex

```diff
@@ -1,7 +1,72 @@
 defmodule Burox.Response.Config do
   @moduledoc false
 
-  @sections_map %{
+  @score_reasons %{
+    "1" => "Nivel de endeudamiento",
+    "4" => "Consulta reciente",
+    "5" => "Pago vencido reciente",
+    "7" => "Cuentas abiertas con morosidad",
+    "9" => "Bajo promedio de antigüedad en créditos abiertos Tipo de crédito con mayor riesgo",
```

Comment:
Qué significa _Bajo promedio de antigëdad en créditos abiertos._
Se puede cambiar o así viene del buró?
```suggestion
    "9" => "Bajo promedio de antigüedad en créditos abiertos. Tipo de crédito con mayor riesgo",
```

Reply:
Asi viene en la documentación del  buró 🤷🏻‍♂️

Reply:
Son las descripciones directas de la documentación del Buró, están raras pero pues ya sabes que su api también esta rara 😬